web/src/main/java/com/futurewei/alcor/web/entity/route/RouteEntry.java

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+
+package com.futurewei.alcor.web.entity.route;
+
+import com.futurewei.alcor.common.entity.CustomerResource;
+import lombok.Data;
+import org.jetbrains.annotations.NotNull;
+
+@Data
+public class RouteEntry extends CustomerResource {
```

Comment:
I found there are two files, both named RouteEntity in this PR. Very unusual. Could you pls double check?

Reply:
One is the RouteEntity, it's used for our previous version of route manager; the other one is RouteEntry, it's used for our new version.

Reply:
Got it. Looked so similar, time to upgrade my glass :-)